src/sync/commits.ts

```diff
@@ -30,37 +29,22 @@ export const getCommits = async (logger: LoggerWithTarget, github: GitHubAPI, gi
 			jiraPayload
 		};
 	}
-	console.log("IM GETTING FROM OCTO");
-	return getCommitsOctoKit(logger, github, gitHubClient, jiraHost, repository, cursor, perPage) 
+	return await getCommitsOctoKit(logger, github, gitHubClient, jiraHost, repository, cursor, perPage);
```

Comment:
This was the await that was missing

Reply:
to make sure this never happens again, please add a specific return type for the function.

Reply:
typed